packages/client/src/templates/register.ts

```diff
@@ -23,6 +23,12 @@ export const certificateBaseTemplate = {
         normal: 'NotoSans-Light.ttf',
         bold: 'NotoSans-Regular.ttf'
       }
+    },
+    fr: {
+      notosans: {
+        normal: 'NotoSans-Light.ttf',
+        bold: 'NotoSans-Regular.ttf'
+      }
```

Comment:
According to Jonathan's comment, the certificate language will be english even the locale is french. So, I guess we don't need to add font here.